### PR TITLE
2019.8 docs

### PR DESCRIPTION
**Pull request by tkfu**

### CODE REVIEW

**Comment by tkfu** [comment]
Remember that if you want to make changes that you want to actually be deployed before the next aktualizr, they should be against the 2019.8-docs branch, not master!

**Comment by patrickvacek** [comment]
> Remember that if you want to make changes that you want to actually be deployed before the next aktualizr, they should be against the 2019.8-docs branch, not master!

Hmmm, true. There's a fine line when it comes to a doc that is mostly only for internal use. I guess just to get used to the workflow, let's try holding off on this and merging docs changes into the latest release docs branch as suggested.

File: docs/ota-client-guide/modules/ROOT/pages/_partials/recommended-steps.adoc [hunk 1]
```diff
@@ -115,3 +115,12 @@ Once you understand how the libaktualizr demo application works, you'll want to
 After you've finished integration and development, you'll need to move disk images from one account to another. For example, you might want to send a development build that you’re happy with to the QA team, or send that build to the deployment team once it’s passed QA.
 
 //  end::deploy-steps[]
+
+//  tag::firstbuild-nextstep[]
+[TIP]
+====
+Once you've build your first image, try adding some new software and xref:ota-client::pushing-updates.adoc[push your update to the OTA Connect server].
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/_partials/recommended-steps.adoc]: `built`?

Comment (tkfu) [reply docs/ota-client-guide/modules/ROOT/pages/_partials/recommended-steps.adoc]: Fixed in 19e0b606.

**Comment by tkfu** [comment]
> Also, we have a lovely merge conflict.

Fixed by 846cbab.

**Comment by patrickvacek** [comment]
@tkfu Any ideas on how to fix this conflict? I thought I might be able to fix it with https://github.com/advancedtelematic/aktualizr/pull/1412 but I was wrong. And when I tried clicking on "Resolve conflicts", it merged master into 2019.8-docs, which was definitely not what I wanted.

**Comment by tkfu** [comment]
> Edit: if github is smart enough, I might be able to merge it manually on the commandline.

Yup, that works just fine, and it's what I usually do when there are merge conflicts and rebase isn't an option.

**Pull request merged**